nexus/tests/output/nexus_tags.txt

```diff
@@ -125,12 +125,12 @@ rack_list                                GET      /v1/system/hardware/racks
 rack_view                                GET      /v1/system/hardware/racks/{rack_id}
 sled_instance_list                       GET      /v1/system/hardware/sleds/{sled_id}/instances
 sled_list                                GET      /v1/system/hardware/sleds
+sled_list_uninitialized                  GET      /v1/system/hardware/uninitialized-sleds
```

Comment:
Should we changed the endpoint to be `/v1/system/hardware/sleds-uninitialized`? This follows a similar pattern where in cases we may have wanted the API to be `/sleds/uninitialized` but can't because of naming conflicts, we use `-` instead.

Reply:
If that's the consensus, I can go ahead and make this change.

Reply:
I agree with Justin. This pattern is not obvious in the API because where it really shows up is the console, e.g., [`/projects`](https://oxide-console-preview.vercel.app/projects) vs. [`/projects-new`](https://oxide-console-preview.vercel.app/projects-new).

There are a few endpoints in the API that are similar, like `/v1/system/silo-quotas` and `/v1/vpc-subnets`, but both of those are hierarchical noun-noun (parent-child) pairs so not quite the same thing. But yes, the basic principle is to line up the names so they're obviously related:

```
/sleds
/sleds-uninitialized
```